app.py

```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
why?

Reply:
my mistake.. didn't notice i deleted "3" from the line